composer.json

```diff
@@ -3,22 +3,26 @@
     "type": "neos-package",
     "description": "This package provides functionality for using Elasticsearch on top of Neos.ContentRepository.Search",
     "homepage": "http://flowpack.org/",
-    "license": [
-        "LGPL-3.0"
-    ],
+    "license": "LGPL-3.0",
     "require": {
         "php": ">=7.0",
-        "flowpack/elasticsearch": "~2.0 || dev-master",
-        "neos/content-repository-search": "~3.0 || dev-master",
-        "neos/flow": "~4.0 || dev-master",
-        "neos/fluid-adaptor": "~1.0 || dev-master"
+        "flowpack/elasticsearch": "^2.0",
+        "neos/content-repository": "^3.0",
```

Comment:
Isn't this required by `neos/content-repository-search` already? See next comment, too…